app/boards/arm/ckp/bt60_v2_defconfig

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: MIT
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_BT60_V2=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# encoder
+CONFIG_EC11=y
+CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+
+CONFIG_USE_DT_CODE_PARTITION=y
+
+CONFIG_ZMK_HID_REPORT_TYPE_NKRO=y
+CONFIG_ZMK_HID_CONSUMER_REPORT_USAGES_BASIC=y
+
```

Comment:
We usually don't override there defaults for boards in ZMK proper. We'll change the defaults for the whole ZMK firmware when ready.